### ISSUE TITLE
Extract Hypertrace Core UI including license concerns

### ISSUE THREAD

**Issue by adriancole** [hypertrace/hypertrace-ui#181]
https://github.com/hypertrace/hypertrace-ui/pull/165 merged Hypertrace Core UI here, which is good because there are less submodules.

However, it appears to have interwoven the core files with the non-core ones. This means Hypertrace Core UI is no longer able to be packaged independently from Hypertrace UI, most notably there's no way to make an Apache licensed (ASL) distribution of the UI anymore.

There will be some work to do in order to pull the files back into a directory so that ASL licensed files can be reviewed deployed and tested independent of Traceable license ones.

**Comment by kotharironak** [comment]
@anandtiwary can we close this?

**Comment by aaron-steinfeld** [comment]
It's not actually done, so that depends on whether packaging HTC separately is still a goal?

**Comment by JBAhire** [comment]
Yes. Let's keep hypertrace-core components into separate directory with Apache licence at root because it's separate licence as of now. 

We might close on licence discussion in near future so if we want to retain HTC let's not make it hard/impossible to achieve.
